Source/Scene/getClippingFunction.js

```diff
@@ -0,0 +1,168 @@
+define([
```

Comment:
Does this belong in `Renderer` or `Scene`?  Probably wherever the plane collection lives.